examples/control-vector-generator/control-vector-generator.cpp

```diff
@@ -756,6 +755,14 @@ int main(int argc, char ** argv) {
     cb_data.n_embd = n_embd;
     int n_prompts = cparams.positive_prompts.size();
 
+    // init ctx_ggml
+    struct ggml_init_params params_ggml = {
+        /*.mem_size   =*/ ggml_tensor_overhead() * n_prompts * n_layers * 4u,
```

Comment:
Where does the hardcoded `4u` come from?

Reply:
It's the 4 std::vector v_pos, v_neg, v_diff,... Probably I allocated more than needed, but I don't really care for now.